brody_lab_to_nwb/interfaces/protocol_info/protocolinfodatainterface.py

```diff
@@ -0,0 +1,61 @@
+"""Authors: Cody Baker and Jess Breda."""
+import numpy as np
+import pandas as pd
+
+from pynwb import NWBFile
+from nwb_conversion_tools.basedatainterface import BaseDataInterface
+from nwb_conversion_tools.utils.json_schema import FilePathType
+
+from .protocol_info_utils import load_nested_mat, make_beh_df
+
+
+class ProtocolInfoInterface(BaseDataInterface):
+    """Conversion class for behavioral info contained in a protocol_info.mat file."""
+
+    @classmethod
+    def get_source_schema(cls):
+        source_schema = dict(
+            required=["file_path"],
+            properties=dict(
+                file_path=dict(
+                    type="string",
+                    format="file",
+                    description="Path to .mat file containing processed data."
+                )
+            ),
+            type="object",
+            additionalProperties=False
+        )
+        return source_schema
```

Comment:
we can generate this, right?

Reply:
We sure can! Since we're using the FilePathTypes from newest version of nwbct. Fixed